mkdocs.yml

```diff
@@ -13,7 +13,8 @@ extra:
 nav:
 - Overview: 'index.md'
 - Specification:
-    - '1.0': 'specification/1.0.md'
+    - Current (draft): 'specification/current.md'
+    - '1.0 (External Link)': https://cds-hooks.hl7.org/1.0/
```

Comment:
Rather than link to 1.0 (and having to add links for each subsequent version), thoughts on linking to the published versions directory
- HL7 Releases (https://cds-hooks.hl7.org/)

Reply:
Great point! Dennis.

Let's do exactly this, once we have more than one release.